clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -368,7 +385,7 @@ def build_model(self,
                 context_tar.add(model_data_path)
                 # From https://stackoverflow.com/a/740854/814642
                 df_contents = six.StringIO(
-                    "FROM {container_name}\nCOPY {data_path} /model/\n".format(
+                    ("FROM {container_name}\nCOPY {data_path} /model/\n"+run_cmd).format(
```

Comment:
Use string formatting instead of appending:

```
"FROM {container_name}\nCOPY {data_path} /model/\n{run_cmd}".format(container_name=base_image, data_path=model_data_path, run_cmd=run_cmd)
```